changes.md

```diff
@@ -1,16 +1,20 @@
 Changes
 =======
 
-master
+2.2.0 (latest release)
```

Comment:
Sketch is a big enhancement and deserves to be highlighted.   Can we add a highlights section to the top like this?:
```
   ###Highlights

   * Introduced a new Sketch API dedicated to 2D planar operations 
   * New constraint types were added to use with the Assembly solver
```

Reply:
Good call. Done.